dashboard/app/subsystems.html

```diff
@@ -13,13 +13,11 @@
 </head>
 <body>
 	{{template "header" .Header}}
-	<h2>The list of subsystems</h2><br>
```

Comment:
I'd keep this, as even with the increased border it's still quite hard to notice what page we're in. It won't hurt to also show the title in the middle of the page, where one expects that :)

Reply:
Done. PTAL.